testgrid/specs/full.yaml

```diff
@@ -1195,16 +1116,19 @@
       version: 1.9.x
     ekco:
       version: latest
-- name: "Upgrade to 1.24 airgap"
+- name: "Upgrade to 1.25 airgap"
   cpu: 6
   installerSpec:
     kubernetes:
-      version: 1.23.x
+      version: 1.24.x
     weave:
       version: latest
-    rook:
-      isBlockStorageEnabled: true
-      version: 1.7.x
+    openebs:
+      version: latest
+      isLocalPVEnabled: true
+      localPVStorageClassName: default
+    minio:
```

Comment:
Haven't we missed `minio` in the `upgradeSpec` down below or is this by design?

Reply:
it's by design, since disableS3 is turned on